recipe/meta.yaml

```
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ build }}
-  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win)) or (cuda_compiler_version != '10.2')]
+  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win))]
```

IIUC this is a CUDA only package. Is that right? If so, think we can make some simplifications

Also curious about the Windows Python 3.10 skip. It looks like that was related to the same CUDA issues ( https://github.com/conda-forge/seekr2_openmm_plugin-feedstock/commit/6114afb8b54816bc1b23a76b110132952245d34f ). Is that right? If so, think we can try relaxing that

Making these changes would leave us with something like this

```suggestion
  skip: true  # [(cuda_compiler or "None") != "None"]
```

---

Yes, it is a CUDA only package, and I believe that indeed the Python 3.10 Windows skip was included for that reason. I'll make these changes and push in a moment...